crates/bevy_window/src/window.rs

```diff
@@ -624,6 +635,17 @@ impl Window {
     }
 }
 
+/// Defines where window should be placed at on creation.
+#[derive(Debug, Clone)]
+pub enum WindowPosition {
+    /// Position will be set by window manager
+    Default,
+    /// Window will be centered at primary monitor
+    Centered,
+    /// Window will be placed at specified position
```

Comment:
```suggestion
    /// The window's center will be placed at the specified position (in pixels)
    ///
    /// (0,0) represents the center of the monitor.
```

Reply:
I don't think that's true. It specifies the top-left corner of the window and (0,0) represents top-left corner of the screen space (not sure if that's the right term. basically the space that contains all of the monitors).
And it was like that before my changes, just to be clear.
I think we *could* make it work as you described, but I think it's not how it usually works, so it would be confusing, at least in my opinion.

Reply:
Yep, if my docs are incorrect please just fix the suggestion :) The point is that this should be explained here.

Reply:
Done!